scripts/User_Management/grade_9_mode.py

```diff
@@ -0,0 +1,41 @@
+from scripts._utils import ssh, pi, utils
```

Comment:
Per issue, this script should be in Miscellaneous

Reply:
fixed as per [6612eae](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/60/commits/6612eae166341b92fd6439632f661f323301261d)